npm_and_yarn/spec/dependabot/npm_and_yarn/file_parser/lockfile_parser_spec.rb

```diff
@@ -121,11 +121,10 @@
       context "that contains an empty version string" do
         let(:dependency_files) { project_dependency_files("pnpm/empty_version") }
 
-        it "raises a DependencyFileNotParseable error" do
-          expect { dependencies }
-            .to raise_error(Dependabot::DependencyFileNotParseable) do |error|
-              expect(error.file_name).to eq("pnpm-lock.yaml")
-            end
+        it "new pnpm dependency path V3.0.0 excludes empty version dependency in the file." do
```

Comment:
Same comment applies here about `it + expected behavior` test descriptions.

Reply:
@bdragon Thanks and updated.